api/handle-form-submit.js

```diff
@@ -1,20 +1,35 @@
 const render = require('../lib/render')
+const axios = require('axios')
 
 module.exports = async (request, response) => {
+  async function renderMessage(status, msg) {
+    const html = await render('submit', { status, msg })
+
+    if (status === 'success') {
+      response.status(200)
+    } else {
+      response.status(500)
+    }
+
+    return response.send(html)
+  }
   // Since every request to this route is a POST request we have
   // access to request.body where all fields can be found
+  const { body, query } = request // Helper function build in by Vercel/Now
 
-  if (!request.body.email) {
-    const html = await render('error', { message: `You have to insert an email` })
+  // Is it a POST request or a GET? Currently only supports POST requests.
+  if (request.method === 'POST') {
+      if (!body.email) {
+        renderMessage('error', { title: 'Geen Email opgegeven', text: 'Vul een geldig email-adres in op het formulier.' })
+      }
 
-    response.status(500)
-    response.send(html)
+      // Validation successful, do this:
+      const POST_REQUEST = await axios.post(query.handler, { data: body })
```

Comment:
Hier dus `node-fetch` gebruiken. Pakketje is een stuk kleiner naast dat we 'm al gebruiken volgens mij? If not, do install. Also, volgens mij kan je die ifjes vermijden door `.then` en `.catch` te gebruiken of een try / catch blok eromheen te gooien.